### PR TITLE
use optionnal basicauth when using http to talk to foreman

### PR DESCRIPTION
**Pull request by arthurlogilab**

### CODE REVIEW

File: etc/foreman.yaml.example [hunk 1]
```diff
@@ -2,9 +2,13 @@
 :proto: https
 :host: foreman.example.com
 :port: 443
+# if using http with ssl certificates
 :ssl_ca: "/var/lib/puppet/ssl/certs/ca.pem"
 :ssl_cert: "/var/lib/puppet/ssl/private_keys/foreman.example.com.pem"
 :ssl_key: "/var/lib/puppet/ssl/certs/foreman.example.com.pem"
+# if using http with username and password instead of https with certicates
+:username: admin
+:password: changeme
```
Comment (stbenjam) [inline etc/foreman.yaml.example]: As they're basically exclusive either SSL *or* username/password auth, would you mind commenting out these 2?

Comment (arthurlogilab) [inline etc/foreman.yaml.example]: done (although I would like to find a way to use login/password auth over https too)

Comment (stbenjam) [inline etc/foreman.yaml.example]: You can still set the URL to https, but you wouldn't need to be providing a key/cert at the same time

**Comment by stbenjam** [comment]
Thanks!

**Pull request merged**